### ISSUE TITLE
paper_input_test fails 'a11y has aria-describedby for character counter'

### ISSUE THREAD

**Issue by ErikGrimes** [dart-lang/polymer_elements#34]
```
expect(input.attributes.containsKey('aria-describedby'), isTrue);
```

```
Expected: true
    Actual: <false>
```

**Comment by jakemac53** [comment]
fixed by https://github.com/dart-lang/polymer_elements/commit/784974541911a465e81e28f1b2d5926e63e0a041
